package.json

```diff
@@ -38,6 +38,7 @@
     "@twogate/ngx-photo-gallery": "^1.4.0",
     "@types/sharedworker": "^0.0.91",
     "angularx-qrcode": "^15.0.1",
+    "animiq-nip76-tools": "file:../../animiq-nip76-tools/dist",
```

Comment:
Is this published yet? Can't test locally without it I think?

Reply:
published nip76-tools just now

Reply:
also note there is some demo code we will want to remove before complete merge.... a custom demo login authguard, with some demo private keys - trying to make it easier for newcomers to preview the concept.
![image](https://user-images.githubusercontent.com/34030518/230431931-ea9c5856-366f-4fb6-9ef8-71c3e0c84809.png)

I can remove that too, but need to make another branch for the demo deployment.

Reply:
@sondreb  - ok - as of this comment everything it does not need was removed and it runs locally using animiq-nip76-tools package version 1.0.5.  - need a git pull.